app/src/main/java/ee/ria/DigiDoc/android/signature/update/Processor.java

```diff
@@ -105,26 +108,30 @@
                 return Observable.just(Result.NameUpdateResult
                         .failure(containerFile, new IOException()));
             } else {
+                String finalName = name;
```

Comment:
Mis selle eesmärk?

Reply:
IDE annab "Variable used in lambda expression should be final or effectively final"

Reply:
aa tõsi, kasutus lambdas ja seda eespool ümberväärtustatud.
Puhtam oleks see 'name' muutuja määramine või Observable stream kasutus panna eraldi meetodisse, aga see ei ole kriitiline.